pkg/networkutils/network.go

```diff
@@ -435,8 +445,26 @@ func useExternalSNAT() bool {
 	return getBoolEnvVar(envExternalSNAT, false)
 }
 
-func randomizeSNAT() bool {
-	return getBoolEnvVar(envRandomizeSNAT, true)
+func randomizeSNAT() snatType {
+	defaultValue := randomHashSNAT
+	defaultString := "hash based random"
+	if strValue := os.Getenv(envRandomizeSNAT); strValue != "" {
+		parsedValue, err := strconv.ParseBool(strValue)
+		if err != nil {
+			if strings.Compare( "prng", strValue) == 0 {
+				return randomPRNGSNAT
+			}
+			log.Error("Failed to parse "+envRandomizeSNAT+"; using default: "+defaultString, err.Error())
+			return defaultValue
+		}
+		// true is equal to hash based random
+		if parsedValue {
+			return randomHashSNAT
+		}
+		// false is equal to sequential
+		return sequentialSNAT
+	}
+	return defaultValue
```

Comment:
Isn't this clearer? Also there's no need to use `strings.Compare` in this case, as we can just compare the strings themselves.
```go
strValue := os.Getenv(envRandomizeSNAT)
if strValue == "" {
	// empty string means default
	return defaultValue
}
if strValue == "prng" {
	// prng means to use --random-fully
	return randomPRNGSNAT
}
parsedValue, err := strconv.ParseBool(strValue)
if err != nil {
	log.Error("Failed to parse "+envRandomizeSNAT+"; using default: "+defaultString, err.Error())
	return defaultValue
}
// true is equal to hash based random
if parsedValue { return randomHashSNAT }
// false is equal to sequential
return sequentialSNAT
```

Reply:
strings.Compare was used in a trivial comparison elsewhere in the code in network.go. I assumed that it was being used for a reason.

Cleaned up the code to be easier to read.